void function 'coap_dtls_handle_timeout' should not return a value

Compiling on OS X gives the following error:

    src/coap_notls.c:86:3: error: void function 'coap_dtls_handle_timeout' should
    not return a value [-Wreturn-type]
      return 0;
      ^      ~

--- a/src/coap_notls.c
+++ b/src/coap_notls.c
@@ -83,7 +83,6 @@ coap_tick_t coap_dtls_get_timeout(coap_session_t *session UNUSED) {
 }
 
 void coap_dtls_handle_timeout(coap_session_t *session UNUSED) {
-  return 0;
 }
 
 int

